### ISSUE TITLE
Magento 2: “Unable to locate element: .page-header .admin-user” when running FTF

### ISSUE THREAD

**Issue by olgamoroz81** [magento/mtf#67]
Hi All,
Can you help me, I am a new in magento2. I tried to run functional test  CreateCategoryEntityTest as is described in guide but I received the error:
Unable to locate element: .page-header .admin-user
![image](https://cloud.githubusercontent.com/assets/25008341/21767992/368245b0-d688-11e6-9cb8-daedf0989137.png)
This screen doesn't have such element

I use Magento ver. 2.1.2

**Comment by lanken** [comment]
Hi @olgamoroz81,
This selector should be on this page, it's present only after successful login. So I've suggested that the reason of failure is that you hadn't logged into the Admin.
Please, check the password and that the backend url specified in the phpunit.xml and in your Magento instance is exact the same. For example, if after login to the Admin manually you have `http://magento.vg/index.php/admin/admin/dashboard/` you should specify in the phpunit.xml `<env name="app_backend_url" value="http://magento.vg/index.php/admin/" />`, if the url is without **index.php** then in phpunit.xml it should be also without it `<env name="app_backend_url" value="http://magento.vg/admin/" />`
See documentation http://devdocs.magento.com/guides/v2.1/mtf/mtf_quickstart/mtf_quickstart_config.html

**Comment by olgamoroz81** [comment]
Seems, my config is correct. But may be I did something wrong.
Can you check please, because I don't really know what to do.
![image](https://cloud.githubusercontent.com/assets/25008341/21799149/1c89c592-d729-11e6-8b7d-768cdbf1eedc.png)
config.xml
![image](https://cloud.githubusercontent.com/assets/25008341/21799170/332dfcc8-d729-11e6-8c46-287d3caf02ba.png)
phpunit.xml
![image](https://cloud.githubusercontent.com/assets/25008341/21799198/47e53636-d729-11e6-86f4-4be5fc914e14.png)

**Comment by lanken** [comment]
Could you, please, provide an url after login to Magento Admin panel.

**Comment by olgamoroz81** [comment]
http://magento2.localhost/admin_2/admin/dashboard/

**Comment by lanken** [comment]
Could you, please, provide a full stack trace of the error?
Also, have you observed the flow of the test? Is it entered login/password to the login form?

**Comment by olgamoroz81** [comment]
"Is it entered login/password to the login form?' - No, throw error

**Comment by lanken** [comment]
Please, provide a full stack trace error.
I couldn't reproduce it locally, the only place wait it waits for the header is `\Magento\Backend\Test\Page\AdminAuthLogin::waitForHeaderBlock`, but it the credentials should be filled previously.

**Comment by olgamoroz81** [comment]
Hi, I found out the problem. It deals with new version selenium server 3.0.1 and new Firefox version 50.  
The problem was following, may be helps to somebody:
1. I have Firefox v50 and selenium server 3.0.1
2. This version of the selenium server ask web-driver, I run it using java -Dwebdriver.gecko.driver="path\to\driver\geckodriver.exe" -jar D:\etc\selenium\selenium-server-standalone-3.0.1.jar
3. But received errors:
Unable to locate element: .page-header .admin-user - in the server trace
moveto did not match a known command - in the test trace
4. Googled this problem I found that Actions Class not working with gecko driver - so I test couldn't login and returned the error about ".page-header .admin-user"
5. After I changed Firefox version to 46 and selenium-server to 2.53 everything works well

Thanks very much @lanken for your help, without you I couldn't found this problem
